doc/script_commands.txt

```diff
@@ -1127,13 +1127,17 @@ In 2015 the tag name was changed to <ITEM> resulting in the following syntax:
 
 	<ITEM>Display Name<INFO>Item ID</INFO></ITEM>
 
+We therefore created a script command "mesitemlink" that allows you to create the correct syntax
+depending on your configured packet version and we recommend that you use this script command
+instead of hardcoding the HTML tags. For more details see the script command documentation later.
```

Comment:
```suggestion
We therefore created script command "mesitemlink" that allows you to create the correct syntax
depending on your configured packet version. We recommend that you use this script command
instead of hardcoding the HTML-like tags. For more details see the documentation for "mesitemlink".
```